src/nrniv/nrnbbcore_write.cpp

```diff
@@ -1,4 +1,4 @@
-#include <../../nrnconf.h>
```

Comment:
nitpick : there is `io/bbcore_write.cpp` and this is `nrnbbcore_write.cpp`. Should we rename `nrnbbcore_write.cpp`  to just `corenrn_write.cpp` ?